### ISSUE TITLE
still heavy battery drain while sleeping

### ISSUE THREAD

**Issue by alofting** [AppHouseKitchen/AlDente-Charge-Limiter#1177]
<img width="477" alt="Screenshot 2024-02-23 at 10 47 21" src="https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/9673307/00dba61a-2eeb-4fde-83db-aa262bfa224d">

I experienced the same problem as in closed issue #91 with my MacBook Pro M1 2021, Sonoma 14.3.1, AlDente Pro 1.24.1; ~30pp drain per 24h in sleep mode until the battery was completely drained the second day sleeping with no screen usage.

AlDente enabled options: 'Stop charging when sleeping', 'Sailing Mode' (5%), 'Heat Protection', 'Updates', 'Launch at Login'

[aldentepro-debug-2024-02-23-11:06:23.pasta.zip](https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/files/14384009/aldentepro-debug-2024-02-23-11.06.23.pasta.zip)

Original power source not connected while machine was "sleeping".

**Comment by MatthiasKerbl** [comment]
Hi @alofting,

Thank you for your bug report, we are currently investigating this issue.
Also, could you please contact our [official support](https://apphousekitchen.com/support/) and mention this GitHub issue? Afterward, we will be able to assess this in more detail.

Best,
Matthias

**Comment by alofting** [comment]
Hi, I have not contacted support yet, but I have done some further testing. I now believe the power drain in sleep mode is caused my external SSD  (SanDisk Extreme Portable SSD 1TB).

I have used this particular drive for over five years. My MacBook Pro is from September 2021. It is only recently that I have noticed this problem. I see that this is a common problem and that it may be unrelated to AlDente. I am not dismissing it entirely yet as alDente interferes with sleep mode.

**Comment by MatthiasKerbl** [comment]
> Hi, I have not contacted support yet, but I have done some further testing. I now believe the power drain in sleep mode is caused my external SSD (SanDisk Extreme Portable SSD 1TB).
> 
> I have used this particular drive for over five years. My MacBook Pro is from September 2021. It is only recently that I have noticed this problem. I see that this is a common problem and that it may be unrelated to AlDente. I am not dismissing it entirely yet as alDente interferes with sleep mode.

Hi @alofting,

Thanks for the update.
If possible, please uninstall AlDente and check if you still experience this issue.

Best,
Matthias

**Comment by alofting** [comment]
Hi Matthias,

Test:
- Sonoma 14.4.1
- AlDente uninstalled (I followed the [instructions](https://apphousekitchen.com/how-to-uninstall-aldente-pro/) - note: I did not find any of the items listed to be in ~/library/)
- hibernatemode 3
Result:
- Battery drained by SSD in sleep mode (after last blue bar on chart below).
￼
<img width="456" alt="Yesterday, 1919" src="https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/9673307/1ee6581c-9f37-40ea-ad87-73f5639df825">

So AlDente is not at fault after all, but more likely the OS, and possibly introduced by Sonoma. I see others are experiencing  problems with their SSDs under Sonoma; device ejected during sleep and long wait when logging in after restart (both of which I have also encountered).

I have tested hibernatemode 25; it puts the SSD to sleep. It takes a bit longer and I do not know if it has other side effects.

My apologies for barking up the wrong tree here.

Best regards,
Arne
